### PR TITLE
Update for proposed configuration system.

### PR DESCRIPTION
**Pull request by windy1**
Configuration refactor pull: https://github.com/SpoutDev/SpoutAPI/pull/93

Signed-off-by: WalkerCrouse (Windwaker) wjcrouse913@aim.com

### CODE REVIEW

File: src/main/java/org/spout/server/util/config/SpoutConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+package org.spout.server.util.config;
+
+import java.io.File;
+import org.spout.api.util.config.Configuration;
+import org.spout.api.util.config.MemoryConfiguration;
+
+public class SpoutConfiguration extends Configuration {
+	
+	private static MemoryConfiguration instance;
+	public static final Nodes Nodes = new Nodes();
```
Comment (Zidane) [inline src/main/java/org/spout/server/util/config/SpoutConfiguration.java]: Make the second Nodes "NODES" as it is essentially read only and final

File: src/main/java/org/spout/server/util/config/SpoutConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	}
+	
+	@Override
+	public void load() {
+		super.load();
+		this.addNodes(Nodes.ADDRESS, Nodes.ALLOW_FLIGHT, Nodes.BANLIST, Nodes.OPS, Nodes.USE_WHITELIST, Nodes.WHITELIST, Nodes.WORLDS, Nodes.WORLD_CONTAINER);
+		this.save();
+	}
+}
\ No newline at end of file
```
Comment (Zidane) [inline src/main/java/org/spout/server/util/config/SpoutConfiguration.java]: Also, be sure to add a new line at the bottom of files :p.

File: src/main/java/org/spout/server/util/config/SpoutConfiguration.java (hunk 1 above)
Comment (windy1) [inline src/main/java/org/spout/server/util/config/SpoutConfiguration.java]: Okay, I just merged the nodes into SpoutConfiguration for static access.

File: src/main/java/org/spout/server/util/config/SpoutConfiguration.java (hunk 2 above)
Comment (windy1) [inline src/main/java/org/spout/server/util/config/SpoutConfiguration.java]: Done.

**Pull request merged**